app-modules/student-data-model/src/Filament/Resources/StudentResource/Pages/ManageStudentPrograms.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+<COPYRIGHT>
+
+    Copyright © 2016-2024, Canyon GBS LLC. All rights reserved.
+
+    Advising App™ is licensed under the Elastic License 2.0. For more details,
+    see https://github.com/canyongbs/advisingapp/blob/main/LICENSE.
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AdvisingApp\StudentDataModel\Filament\Resources\StudentResource\Pages;
+
+use Laravel\Pennant\Feature;
+use Filament\Resources\Pages\ManageRelatedRecords;
+use AdvisingApp\StudentDataModel\Filament\Resources\StudentResource;
+use AdvisingApp\BasicNeeds\Filament\Resources\BasicNeedsProgramResource\RelationManagers\ProgramRelationManager;
+
+class ManageStudentPrograms extends ManageRelatedRecords
+{
+    protected static string $resource = StudentResource::class;
+
+    protected static string $relationship = 'basicNeedsPrograms';
+
+    protected static ?string $navigationIcon = 'heroicon-o-square-3-stack-3d';
+
+    protected static ?string $breadcrumb = 'Programs';
+
+    protected static ?string $title = 'Programs';
+
+    public static function getNavigationLabel(): string
+    {
+        return 'Programs';
+    }
+
+    public function getRelationManagers(): array
+    {
+        return [ProgramRelationManager::class];
+    }
+
+    public static function canAccess(array $parameters = []): bool
+    {
+        if (Feature::active('manage-program-participants')) {
+            return true;
```

Comment:
This should return the parent `canAccess`

Reply:
Hi @Orrison ,
Directly parent::canAccess() is giving forbidden errors. So, I passed the $parameters into that to fix the forbidden issue.

Thank You!